functions/src/index.ts

```diff
@@ -232,6 +232,123 @@ function cleanAverages(averages: SensorReading[]): CleanedReadings {
   return new CleanedReadings(latitude, longitude, cleanedAverages);
 }
 
+/**
+ * Computes the AQI for PM 2.5 given the appropriate AQI breakpoints. in most use cases, this function will be called from a function which knows those breakpoints.
+ *
+ * Adapted from EPA function available at https://www.airnow.gov/sites/default/files/custom-js/conc-aqi.js
+ *
+ * @param pm25Concentration - the PM 2.5 concentration to calculate the AQI for
+ * @param lowConcentrationBreakpoint - the low breakpoint for PM 2.5 that the concentration falls within
+ * @param highConcentrationBreakpoint - the high breakpoint for PM 2.5 that the concentration falls within
+ * @param lowIndexBreakpoint - the low breakpoint for AQI that the AQI will fall between
+ * @param highIndexBreakpoint - the high breakpoint for AQI that the AQI will fall between
+ */
+function _indexCalculation(
```

Comment:
I think we should move this function and aqiFromPm25 into a separate file and then import them, since both functions are essentially just number crunching, and they don't require anything from the index.ts file

Reply:
Done!